jetbrains-rider/src-232+/software/aws/toolkits/jetbrains/utils/OpenSolutionDirName.kt

```diff
@@ -0,0 +1,6 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.utils
+
+const val OPEN_SOLUTION_DIR_NAME: String = "testData/solutions/SamHelloWorldApp/SamHelloWorldApp.sln"
```

Comment:
This const along with the function openSolutionFile could be in a single util file?

Reply:
Resolved